conf/defaults.ini

```diff
@@ -105,6 +105,11 @@ log_queries =
 # For "mysql", use either "true", "false", or "skip-verify".
 ssl_mode = disable
 
+# Database drivers may support different transaction isolation levels.
```

Comment:
Can we describe it so that it's clear this is only currently supported for MySQL? Checked postgres and seems you cannot set this through the use of connection string, [ref](https://stackoverflow.com/questions/62649971/set-default-global-postgres-transaction-isolation-level).